### ISSUE TITLE
How can you rename Repeater Add Item button?

### ISSUE THREAD

**Issue by CodeCurosity** [pojome/elementor#6699]
I would like to change repeater "Add Item" to "Add New Image", is there any setting by default?

**Comment by bainternet** [comment]
@CodeCurosity 

Currently, this feature is not supported, but we will look into adding it.

Thanks for the feedback.

**Comment by CodeCurosity** [comment]
I don't think it is such a big feature to implement, its been well over 6 months and nobody addressed this issue?

**Comment by r00ten** [comment]
Did you ever add this feature?
